index.md

```diff
@@ -1,3 +1,4 @@
+# This is a big change!
```

Comment:
Does this heading really below here? I recommend removing this heading entirely.